src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -314,10 +321,10 @@ valid_entropy_sources = @DEFVALIDENTROPYSOURCES@
 # the OCI spec passed to the runtime.
 #
 # You can create a rootfs with hooks by customizing the osbuilder scripts:
-# https://github.com/kata-containers/kata-containers/tree/main/tools/osbuilder
```

Comment:
This was fixed by [the commit](https://github.com/kata-containers/kata-containers/pull/2261).
Cloud you please put it back?

Reply:
Thank you @ManaSugi , I've updated the code according to your comments.